### DISCUSSION TITLE
How to configure “no openssl required”

### DISCUSSION THREAD

**Discussion by luzizheng** [eProsima/Fast-DDS#5528] [Q&A] [answered]
Can openssl as well as the security module be used as configuration items for conditional compilation?

**Comment by cferreiragonz** [comment, marked as answer]
Yes! OpenSSL is required by the Security Module and TLS. Disabling these two options will allow compilation without OpenSSL, which can be achieved with:
`colcon build --packages-select fastdds --cmake-args -DSECURITY=OFF -DNO_TLS=ON`
More info about Fast DDS compilation flags can be found [here](https://fast-dds.docs.eprosima.com/en/latest/installation/configuration/cmake_options.html#general-options).
